#/***********************************************************************
# * Licensed Materials - Property of IBM 
# *
# * IBM SPSS Products: Statistics Common
# *
# * (C) Copyright IBM Corp. 1989, 2014
# *
# * US Government Users Restricted Rights - Use, duplication or disclosure
# * restricted by GSA ADP Schedule Contract with IBM Corp. 
# ************************************************************************/

# NLS_ENCODING=UTF-8
# IBMUPYTH


# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2014-05-21 17:51+China Standard Time\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"


#: SPSSINC_MERGE_TABLES.py:25
msgid "This module requires at least SPSS Statistics 17"
msgstr ""
"このモジュールには SPSS Statistics 17 以上が必要です"

#: SPSSINC_MERGE_TABLES.py:266
msgid "---"
msgstr ""
"---"

#: SPSSINC_MERGE_TABLES.py:323
msgid "Function reference %s not valid"
msgstr ""
"関数リファレンス %s が有効ではありません"

#: SPSSINC_MERGE_TABLES.py:344
msgid "SpssClient.StartClient failed."
msgstr ""
"SpssClient.StartClient に失敗しました。"

#: SPSSINC_MERGE_TABLES.py:423
msgid "Incorrect number of table objects specified: %s"
msgstr ""
"指定したテーブル オブジェクト数 %sが不正です"

#: SPSSINC_MERGE_TABLES.py:447
msgid "Exactly two table numbers or OMS subtypes must be specified or a requested table not found."
msgstr ""
"2 つのテーブル番号または OMS サブタイプを指定する必要があります。指定しない場合、必須テーブルは見つかりません。"

#: SPSSINC_MERGE_TABLES.py:474
msgid "At least two table numbers must be specified."
msgstr ""
"2 つ以上のテーブル番号を指定する必要があります。"

#: SPSSINC_MERGE_TABLES.py:505
msgid "mode must be merge or replace"
msgstr ""
"結合モードまたは置換モードでなければなりません。"

#: SPSSINC_MERGE_TABLES.py:514
msgid "A specified item is not a pivot table or does not exist."
msgstr ""
"指定した項目がピボット テーブルでないか、存在しません。"

#: SPSSINC_MERGE_TABLES.py:559
msgid "Only the first layer has been merged from the secondary table."
msgstr ""
"セカンダリ テーブルの最初の層だけが結合されています。"

#: SPSSINC_MERGE_TABLES.py:612
msgid ""
"Tables cannot be merged: The secondary table labels are not unique.\n"
"First duplicate row and column labels after transformation:\n"
"%s\n"
"\n"
"%s"
msgstr ""
"テーブルを結合できません。 セカンダリ テーブルのラベルが一意ではありません。\n"
"変換後、まず行ラベルおよび列ラベルを複製してください。\n"
"%s\n"
"\n"
"%s"

#: SPSSINC_MERGE_TABLES.py:739 SPSSINC_MERGE_TABLES.py:748
#: SPSSINC_MERGE_TABLES.py:848 SPSSINC_MERGE_TABLES.py:852
#: SPSSINC_MERGE_TABLES.py:859 SPSSINC_MERGE_TABLES.py:863
msgid "Statistics"
msgstr ""
"統計"

#: SPSSINC_MERGE_TABLES.py:794
msgid "A row label level is out of range.  Labels: %s, Index: %s"
msgstr ""
"行ラベルのレベルが範囲外です。  ラベルは %s、インデックスは %s です。"

#: SPSSINC_MERGE_TABLES.py:811
msgid "A column label level is out of range.  Labels: %s, Index: %s"
msgstr ""
"列ラベルのレベルが範囲外です。  ラベルは %s、インデックスは %s です。"



